src/lib/statistics/statistics_objects/counting_quotient_filter.cpp

```diff
@@ -126,6 +123,34 @@ bool CountingQuotientFilter<ElementType>::is_full() const {
   return load_factor() > 0.99f;
 }
 
+template <typename ElementType>
+Cardinality CountingQuotientFilter<ElementType>::estimate_cardinality(
+    const PredicateCondition predicate_condition, const AllTypeVariant& variant_value,
+    const std::optional<AllTypeVariant>& variant_value2) const {
+  if (predicate_condition == PredicateCondition::Equals) {
+    if (does_not_contain(variant_value)) return 0;
+    return static_cast<Cardinality>(count(variant_value));
+  } else {
+    // Cannot use CQFs for cardinality estimation beyond equality predicates
+    return 0;

```

Comment:
selectivity of 1 should be default, this 0 was misplaced here. Sadly we cannot make that a constant across statistics object, since it requires a member function call
```c++

    // Cannot use CQFs for cardinality estimation beyond equality predicates
    return static_cast<Cardinality>(count(variant_value));
```